src/python/convert_numpy_features_to_protos.py

```diff
@@ -0,0 +1,84 @@
+import numpy as np
+from pathlib import Path
+import argparse
+import protos_io
+
+import protos.localization_protos_pb2 as loc_protos
+
+
+def convert_to_protos(features, feature_type):
+    """Converts the numpy nd array to the features protos.
+
+    Args:
+        features (numpy.ndarray): feature vectors to be converted.
+                                Expected size NxD, where N is
+                                number of features and D is number
+                                of dimensions.
+    Returns:
+        [image_sequence_localizer.Feature]: list of feature protos.
+                                            For details check
+                                            localization_protos.proto file.
+    """
+    protos = []
+    assert len(features.shape) == 2
```

Comment:
nit: Maybe a check with a more friendly message would be better here? Not sure about it, but keep thinking if I use the script and it hits this assert, what will be printed and will I understand what I did wrong?